### DISCUSSION TITLE
get the total charging after each loading process

### DISCUSSION THREAD

**Discussion by SebbyIU** [goecharger/go-eCharger-API-v2#151] [Q&A]
Hi,

i´m using the local http api.
I want to write a little script in Python to store the total charging after each loading process in a mySQL database.

I get the total charging over all with the api call "eto". Is there a way to store this value after each loading process? For example is there a way to get the last "charging stop time"?

Thanks in advance.

**Comment by kalaws** [comment]
The `wh` key reports "energy in Wh since car connected," according to API description.
